docs/executor-k6.md

```diff
@@ -0,0 +1,55 @@
+# K6 Performance Tests
+
+[K6](https://k6.io/docs/) Grafana k6 is an open-source load testing tool that makes performance testing easy and productive for engineering teams. k6 is free, developer-centric, and extensible.
+
+Using k6, you can test the reliability and performance of your systems and catch performance regressions and problems earlier. K6 will help you to build resilient and performant applications that scale.
+
+k6 is developed by Grafana Labs and the community.
+
+## Running a K6 test
+
+K6 is integral part of Testkube. Testkube K6 executor is installed by default in recent Testkube installation. To run K6 test in Testkube you need to create Test. 
```

Comment:
K6 is integral part of Testkube. The Testkube k6 executor is installed by default during the Testkube installation. To run a k6 test in Testkube, you need to create a Test.